lighthouse-cli/test/fixtures/source-maps/source-maps-tester.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<!--
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+-->
+
+<html>
+<head>
+  <title>Source maps tester</title>
+  <meta charset="utf-8">
+</head>
+<body>
+  <!-- Note: none of these source maps are _actually_ maps of the 'generated' code here. -->
+  <!-- It's whatever I typed into https://dopiaza.org/tools/datauri/index.php -->
+  <script>
+    const a = 1;
+    // Bad inline json:
+    // {{}
+    //# sourceMappingURL=data:application/json;charset=utf-8;base64,e3t9
+  </script>
+
+  <script>
+    const a = 1;
+    // Good inline json:
+    // {
+    //   "version" : 3,
+    //   "file": "out.js",
+    //   "sourceRoot" : "",
+    //   "sources": ["foo.js", "bar.js"],
+    //   "names": ["src", "maps", "are", "fun"],
+    //   "mappings": "AAgBC,SAAQ,CAAEA"
+    // }
+    //# sourceMappingURL=data:application/json;charset=utf-8;base64,ew0KICAidmVyc2lvbiIgOiAzLA0KICAiZmlsZSI6ICJvdXQuanMiLA0KICAic291cmNlUm9vdCIgOiAiIiwNCiAgInNvdXJjZXMiOiBbImZvby5qcyIsICJiYXIuanMiXSwNCiAgIm5hbWVzIjogWyJzcmMiLCAibWFwcyIsICJhcmUiLCAiZnVuIl0sDQogICJtYXBwaW5ncyI6ICJBQWdCQyxTQUFRLENBQUVBIg0KfQ==
+  </script>
+
+  <script src="bundle.js"></script>
+
+  <script src="bundle-map-500.js"></script>
```

Comment:
not sure what `bundle-map-500` means. Maybe like `valid-bundle.js` and `invalid-source-map-url-bundle.js` or whatever so they start to be self documenting?

Reply:
would also be good to annotate in here with successes and failures

Reply:
renamed